.buildkite/beta-builds.yml

```
@@ -5,7 +5,7 @@
 common_params:
   # Common plugin settings to use with the `plugins` key.
   - &common_plugins
-    - automattic/bash-cache#2.11.0
+    - automattic/a8c-ci-toolkit#2.14.0
```

Version [2.15.0](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/releases/tag/2.15.0) has been released in the meantime.
```suggestion
    - automattic/a8c-ci-toolkit#2.15.0
```

---

Ahh yes, thanks! I didn't release there was already a newer version released until a few PRs later...